src/pages/home.js

```diff
@@ -38,16 +41,29 @@ const ExportType = {
   TEXT: 'text',
 };
 
+const LatexDelimiter = {
+  DOLLAR: 'dollar',
+  BRACKET: 'bracket',
+};
+
+const DocumentFormat = {
+  BLOCK: 'block',
+  INLINE: 'inline',
+};
+
+const DocumentColor = {
+  LIGHT: 'light',
+  DARK: 'dark',
+};
+
```

Comment:
As my comment in `src/components/home/setting-modal/helpers.js`, you could consider to collect the module related to displayConfig into one lib namespace.